mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent {
+        docker {
+            image 'rocm/mlir:miopen_test'
+            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+            label 'gfx908'
+        }
+    }
+    stages {
+        stage('Environment') {
+            steps {
+                sh 'cat /etc/os-release'
+                sh '/opt/rocm/bin/rocm-smi'
+            }
+        }
+        stage('Build libMLIRMIOpen') {
+            steps {
+                checkout scm
+                sh '''
+                    mkdir -p build && cd build
+                    /usr/local/bin/cmake -G "Unix Makefiles" ../llvm \
+                         -DLLVM_ENABLE_PROJECTS="mlir;lld" \
+                         -DCMAKE_BUILD_TYPE=Release \
+                         -DBUILD_SHARED_LIBS=OFF \
+                         -DLLVM_BUILD_LLVM_DYLIB=OFF \
+                         -DLLVM_ENABLE_TERMINFO=OFF
+                    make -j$(nproc) libMLIRMIOpen
```

Comment:
Could you go back to home dir from here?

Reply:
I don't have to. The control automatically goes to the llvm-project-mlir directory at the beginning of each stage.